### PR TITLE
media-sound/mpdscribble: add 9999 version

### PR DESCRIPTION
**Pull request by soredake**
updated ebuild for 0.22 version
updated libmpdclient ebuilds

### CODE REVIEW

File: media-libs/libmpdclient/libmpdclient-2.10.ebuild [hunk 1]
```diff
@@ -18,6 +18,7 @@ DEPEND="doc? ( app-doc/doxygen )"
 
 src_prepare() {
 	sed -e "s:@top_srcdir@:.:" -i doc/doxygen.conf.in
+	eapply_user
```
Comment (SoapGentoo) [inline media-libs/libmpdclient/libmpdclient-2.10.ebuild]: don't call `eapply_user`, instead, call `default`, and call it before the sed

File: media-libs/libmpdclient/libmpdclient-2.10.ebuild [hunk 2]
```diff
@@ -18,6 +18,7 @@ DEPEND="doc? ( app-doc/doxygen )"
...
 }
 
 src_configure() {
```
Comment (SoapGentoo) [inline media-libs/libmpdclient/libmpdclient-2.10.ebuild]: remove `--docdir` from econf (part of `EAPI=6`)

**Comment by soredake** [comment]
Done.

**Comment by SoapGentoo** [comment]
@soredake can you please rebase on latest master to get rid of the CI error?

File: media-sound/mpdscribble/mpdscribble-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	!curl? ( net-libs/libsoup:2.4 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_configure() {
+	local myclient=soup
+	use curl && myclient=curl
+	econf \
+		--with-http-client=${myclient} \
+		--docdir="${EPREFIX}"/usr/share/doc/${PF}
```
Comment (SoapGentoo) [inline media-sound/mpdscribble/mpdscribble-9999.ebuild]: again, get rid of `--docdir`

File: media-sound/mpdscribble/mpdscribble-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	curl? ( net-misc/curl )
+	!curl? ( net-libs/libsoup:2.4 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_configure() {
+	local myclient=soup
+	use curl && myclient=curl
+	econf \
+		--with-http-client=${myclient} \
```
Comment (SoapGentoo) [inline media-sound/mpdscribble/mpdscribble-9999.ebuild]: drop the `myclient` variable, and just do
`--with-http-client=$(usex curl curl soup) \`

**Comment by soredake** [comment]
Rebased and updated.

**Pull request closed without merging**